src/components/Chat/ChatMessage.tsx

```diff
@@ -78,7 +78,7 @@ class ChatMessage extends UIComponent<Extendable<IChatMessageProps>, any> {
 
   static defaultProps = {
     accessibility: ChatMessageBehavior as Accessibility,
-    as: 'li',
+    as: 'div',
```

Comment:
have take a look on how `children` prop is handled for this component - seems that there is a problem with that, as `ElementType` will be always a wrapper for the content provided as `children`. Could we address this problem in this PR, or you think it would be better to file an issue for that?